WireDomain/Sources/WireDomain/Repositories/Conversations/LocalStore/ConversationLocalStore.swift

```diff
@@ -1236,6 +1236,41 @@ public final class ConversationLocalStore: ConversationLocalStoreProtocol {
         }
     }
 
+    // MARK: - Backup / Restore
+
+    public func totalConversationCountForBackup() async throws -> Int {
```

Comment:
Since you have a new specific store for backups, can we revert these changes?

Reply:
done